openbas-api/src/main/java/io/openbas/service/RoleService.java

```diff
@@ -70,4 +71,24 @@ public void deleteRole(@NotBlank final String roleId) {
 
     roleRepository.deleteById(roleId);
   }
+
+  /**
+   * Get a set of capabilities as input and return a set containing the input + their parent
+   *
+   * @param capabilitiesInput
+   * @return
+   */
+  @VisibleForTesting
```

Comment:
Not a fan of this annotaiton at all. At the same time, the method needs to be tested and doens't need to be public, so.... Let's go with it none the less.